Bookstore: styling

khanh-devos opened the pull request:
Tasks in this activity:
- Do the styling based on [Zeplin's design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378).
- Add react-circular-progress-bar to build the middle column
- Leverage CSS while styling.

khanh-devos committed:
Do styling the app following Zeplin project.
- Adding react-circular-progressbar to build the middle column.
- Integrate the delete button into the 'remove' button at the first column.
- Leverage CSS to do the styling only.

khanh-devos committed:
modify MIT.md

iambenkis commented on src/App.js:
```diff
@@ -19,22 +20,36 @@ function App() {
         <header className="header">
-          <h1>
-            SECRET FOREST
+          <h1 className="header-h1">
+            Bookstore CMS
           </h1>
           <ul className="header-ul">
             <li>
-              <Link className="header-ul-link" to="/">
-                Books
+              <NavLink
```
- [ ] Kindly make that your design is exactly the same as in the [given design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) 😄  as it is a part of the requirements 👍. Make sure `color` , `padding`.  `margin` , etc , are exactly the same as in the given design 👍  

### Navbar
| Your Design  | Given design |
| :---: | :---: |
| ![Capture](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/1feaffc6-5467-475e-ab10-cd5d034c58d8) | ![Capture 1](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/ebae1bb9-5ec1-46e7-872c-f13cbcc332c3) |

### BookList
| Your Design  | Given design |
| :---: | :---: |
| ![Capture33](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/d512d8ee-a0bc-4f4b-8acb-881db738cbde) | ![Capture33](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/fbde0131-0276-4314-a74f-b5eefaf4a75e) |

### Add book form
| Your Design  | Given design |
| :---: | :---: |
| ![Capture4](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/1f0dedbb-e47d-47af-9d06-d6d3886480e8)  | ![Capture44](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/72d7ea33-c66f-42b2-aef0-13afd2734d85)
 |

khanh-devos replied:
Hi @iambenkis ,
Thank you for your quick response.

iambenkis reviewed (changes requested):
#### Hi @khanh-devos 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

![giphy](https://user-images.githubusercontent.com/25227022/169391888-8dd8108f-0d42-4e54-87cc-a3f9fd392dbc.gif)


## Highlights

✔️  No linter errors
✔️ Used Gitflow  

## Suggested changes

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

khanh-devos committed:
Fix paddings among items following exactly the Zeplin design.
- Fix the padding top, left, and right of the body.
- Fix the padding of the button "Update Progress".

Alfredbis29 reviewed (approved):
# STATUS: APPROVED   
 ## Hi  @khanh-devos 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Good Job :+1:
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :

![](https://camo.githubusercontent.com/54a5d803a138b5a1a1242a607e4c1192c852cab8c591c1ddcaa33d06b51560f6/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f686c516559335062684833692f67697068792e676966)
 
 ## Optional suggestions

- [x] Keep doing the Great work :clinking_glasses: 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

khanh-devos commented:
Hi @Alfredbis29 
Thank you very much.
